### PR TITLE
migrating setup to yarn & volta

### LINKED ISSUE #40: Use yarn and volta
Let's provide instructions on using volta and yarn, and how it helps with automatic node version switching

### PR DESCRIPTION
**Pull request by Mir-SA**
### What is the change?

Migrates setup to yarn and volta for package management and version switching.
Closes #40 

### Is it bug? No

- Steps to repro
- Expected - Provide instructions for setup with yarn & volta
- Actual - Provide instructions for setup with yarn & volta

### \*Dev Tested?

- [x] Yes
- [ ] No

#### Platforms

- [x] Web

#### Browsers

- [x] Chrome
- [ ] Safari
- [ ] Firefox

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,5 +75,9 @@
   "engines": {
     "node": ">=14"
   },
-  "sideEffects": false
-}
+  "sideEffects": false,
+  "volta": {
+    "node": "14.18.2",
```
Comment (harshith-venkatesh) [inline package.json:80]: Could this be upgraded to 16 version please

**Review by harshith-venkatesh — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Mir-SA) [reply package.json]: @harshith-venkatesh I've made the change!
please check it out.

**Review by harshith-venkatesh — APPROVED** [review]

**Comment by harshith-venkatesh** [comment]
# Yarn Command Reference

##### `yarn`

Installs all `dependencies` listed in the root `package.json`

Could you add this as well please.
This is currently missed.
And good to know that you have self reviewed and removed `run`

**Review by ankushdharkar — APPROVED** [review]

**Comment by ankushdharkar** [comment]
Thank you @Mir-SA! This is going to help us become more productive

**Comment by Mir-SA** [comment]
> # Yarn Command Reference
> ##### `yarn`
> Installs all `dependencies` listed in the root `package.json`

not sure if I can make that change since it is merged now...

> Could you add this as well please. This is currently missed. And good to know that you have self reviewed and removed `run`

I made that change as suggested by Ankush!

**Pull request merged**